src/main/webapp/transcript.js

```diff
@@ -56,27 +57,28 @@ function addMultipleTranscriptLinesToDom(transcriptLines) {
 }
 
 /**
- * Creates an <li> element containing {@code transcriptLine}'s text, start time,
- * and end time and appends it to {@code ulElement}.
+ * Creates an <li> element containing {@code transcriptLine}'s text, start
+ * time, and end time and appends it to {@code ulElement}.
  */
 function appendTextToList(transcriptLine, ulElement) {
-  const liElement = document.createElement('li');
-  const startDate = new Date(transcriptLine.start);
-  const endDate = new Date(transcriptLine.end);
-  const startTimestamp = `${startDate.getHours()}:${startDate.getMinutes()}:${
-    startDate.getSeconds()}`;
-  const endTimestamp =
-    `${endDate.getHours()}:${endDate.getMinutes()}:${endDate.getSeconds()}`;
+  const startTimestamp = timestampToString(transcriptLine.start);
+  const endTimestamp = timestampToString(transcriptLine.end);
   const timestamp = `${startTimestamp} - ${endTimestamp}`;
 
-  appendParagraphToContainer(timestamp, liElement, ['mx-auto']);
-  liElement.classList.add('d-flex', 'flex-row', 'justify-content-between');
-  appendParagraphToContainer(transcriptLine.content, liElement, ['mx-auto']);
-  liElement.appendChild(document.createElement('hr'));
-  ulElement.appendChild(liElement);
+  const contentDivElement = document.createElement('div');
+  contentDivElement.classList.add('d-flex', 'flex-row', 'mb-1');
+  appendParagraphToContainer(
+      timestamp, contentDivElement, ['justify-content-start', 'mb-1']);
+  appendParagraphToContainer(
+      transcriptLine.content, contentDivElement, ['ml-4', 'mb-1']);
 
-  liElement.startDate = startDate;
-  liElement.endDate = endDate;
+  const liElement = document.createElement('li');
+  liElement.classList.add('align-self-center', 'mb-2');
+  liElement.appendChild(contentDivElement);
+  const hrElement = document.createElement('hr');
+  hrElement.classList.add('my-1', 'align-middle', 'mr-5');
+  liElement.appendChild(hrElement);
+  ulElement.appendChild(liElement);
```

Comment:
This section is a little hard to follow.  If the transcript styling gets more complicated (like when we add references/icons for comments) we might want to consider making this a customElement using a template

Reply:
I created an issue (#122) for this and will get to it later. Thanks for the tip!